.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 9.x, 10.x, 11.x, 12.x, 14.x, 15.x, 16.x]
```

Comment:
Any particular reason we're dropping this? If it's not incompatible with anything we're trying to achieve I'd say no reason, right?

Btw:
- 12 and 14 are not EOL yet, they're on maintenance release schedules.
- 18 is an LTS so should probably be added.

Reply:
easier on us to drop support 

> 12 and 14 are not EOL yet

12 is EOL in 3 days man 😂 14 is kept in.

18 isn't the current LTS, I can add it in, don't mind